app/database/server/models/channel.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import Model, {Associations} from '@nozbe/watermelondb/Model';
+import {MM_TABLES} from '@constants/database';
+import field from '@nozbe/watermelondb/decorators/field';
+import children from '@nozbe/watermelondb/decorators/children';
+import Draft from '@typings/database/draft';
+import GroupsInChannel from '@typings/database/groups_in_channel';
+import PostInChannel from '@typings/database/post_in_channel';
+import MyChannelSettings from '@typings/database/my_channel_settings';
+import MyChannel from '@typings/database/my_channel';
+import ChannelInfo from '@typings/database/channel_info';
+import immutableRelation from '@nozbe/watermelondb/decorators/immutableRelation';
+import relation from '@nozbe/watermelondb/decorators/relation';
```

Comment:
Please keep the imports in order...

We should document this, the order should be always alphabetical based on the module not the variable name and then divided in sections with a new line between them:
1. Packages in node_modules
2. Reference to other files in the project
3. Other files in the same directory

Reply:
Yes sure. I will amend the imports

Reply:
Done